kustomize/commands/edit/fix/fix_test.go

```diff
@@ -130,19 +365,26 @@ labels:
     foo: bar
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
-`)
-	fSys := filesys.MakeFsInMemory()
-	testutils_test.WriteTestKustomizationWith(fSys, kustomizationContentWithOutdatedCommonLabels)
-	cmd := NewCmdFix(fSys, os.Stdout)
-	assert.NoError(t, cmd.RunE(cmd, nil))
+`,
+		},
+	}
+	for _, test := range tests {
```

Comment:
Can we move the loop body into the body of a function passed to `t.Run()`? [The Run method](https://pkg.go.dev/testing#T.Run) runs its parameter function as a sub-test, which will display the results as such and isolate failures between sub-tests. We can also pass in `test.name`.

Reply:
I fixed at https://github.com/kubernetes-sigs/kustomize/pull/4733/commits/7c2e8845ad1127c5dc52cc7c8450ac3bb2ddaa46